elephant/test/test_spike_train_dissimilarity.py

```diff
@@ -47,6 +47,7 @@ def setUp(self):
         self.st23 = StationaryPoissonProcess(rate=30 * Hz, t_start=0 * ms,
                                              t_stop=1000 * ms
                                              ).generate_spiketrain()
+        self.st24 = SpikeTrain([0.1782, 0.2286, 0.2804, 0.4972, 0.5504], units='s',t_stop=4.0)
```

Comment:
As suggested by the dedicated regression test, the spike train can be initialized locally in the function, which facilitates code reading.